server/openapi_server/openapi/openapi.yaml

```diff
@@ -422,6 +424,27 @@ paths:
       tags:
       - Grant
       x-openapi-router-controller: openapi_server.controllers.grant_controller
+  /healthcheck:
+    get:
+      description: Get information about the health of the service
+      operationId: get_health_check
+      responses:
```

Comment:
The 500 response is missing. https://github.com/nlpsandbox/nlpsandbox-schemas/blob/develop/openapi/commons/paths/healthCheck.yaml